maple-sdk/introduction.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+`maple-js` is a comprehensive and user-friendly JavaScript SDK for Maple Protocol. It is designed to enable seamless interaction with Maple Protocol's smart contracts on Ethereum. This SDK equips developers with the essential tools to integrate with the Maple Protocol ecosystem, simplifying the process of connecting to the blockchain, managing accounts, submitting transactions, and handling events generated by smart contracts.
+
+Utilizing the JavaScript SDK for Maple Protocol allows developers to efficiently create robust and secure dApps that leverage the full potential of Maple Protocol's smart contracts, fostering innovation and driving the adoption of decentralized finance solutions.
+
+Whether you are building backend services, frontend applications, or a combination of both, the Maple SDK provides the tools you need to create a seamless user experience.
+
+# Exports
```

Comment:
Going from the Introduction straight to Exports seems a bit hasty. Can we explain in any more detail either what the SDK contains and/or explain what the Exports list is?